relayer/client.go

```diff
@@ -316,12 +325,20 @@ func MsgUpdateClient(
 	eg.Go(func() error {
 		return retry.Do(func() error {
 			var err error
+			src.log.Debug(
+				"Query IBC header",
+				zap.String("chain_id", src.ChainID()),
+				zap.String("client_id", src.ClientID()),
+				zap.Int64("height", int64(dstClientState.GetLatestHeight().GetRevisionHeight())+1),
```

Comment:
Why do we need to do `dstClientState.GetLatestHeight().GetRevisionHeight() + 1` here instead of just using `dstClientState.GetLatestHeight().GetRevisionHeight()`?

Reply:
ahh thanks, that makes sense to me